web/src/App.jsx

```
@@ -4408,34 +4459,39 @@ function App() {
+                {isLoadingContacts ? (
+                  Array.from({ length: 10 }).map((_, i) => <ContactSkeleton key={i} />)
+                ) : (
+                  <>
+                    {contactListElements}
+                    {filteredDeviceContacts.length > contactListLimit && (
+                      <button
+                        className="secondary-btn"
+                        style={{ marginTop: '16px', width: '100%' }}
+                        onClick={() => setContactListLimit(prev => prev + 50)}
+                      >
+                        Show more contacts
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)